### PR TITLE
Tickets/dm 7436

### PR DESCRIPTION
**Pull request by fjammes**

### CODE REVIEW

File: admin/tools/qserv-install.sh [hunk 1]
```diff
@@ -7,8 +7,8 @@ DIR=$(cd "$(dirname "$0")"; pwd -P)
 
 # Default values below may be overidden by cmd-line options
 MODE="internet mode"
-NEWINSTALL_URL="https://raw.githubusercontent.com/lsst/lsst/master/scripts/newinstall.sh"
-VERSION="-t qserv"
+NEWINSTALL_URL="https://raw.githubusercontent.com/lsst/lsst/12.0/scripts/newinstall.sh"
```
Comment (vaikuntht) [inline admin/tools/qserv-install.sh]: This is a hard coded link, is it the right one to use? I thought ``master`` was the latest one.

Comment (fjammes) [inline admin/tools/qserv-install.sh]: LSST documentation recommend using the release link:
https://pipelines.lsst.io/install/newinstall.html
master could be broken as it is development version, isn't it?

Comment (timj) [inline admin/tools/qserv-install.sh]: Master is not meant to be broken as such. LSST policy is that master is always fine. Whether you want the version associated with a specific release or the current "best" is a different question. For example, 12.0 does not support python 3 but master does.

Comment (fjammes) [inline admin/tools/qserv-install.sh]: Thanks for your review :-)

**Pull request merged**